src/pyscaffold/shell.py

```diff
@@ -47,6 +51,10 @@ def run(self, *args, **kwargs) -> subprocess.CompletedProcess:
         """Execute command with the given arguments via :obj:`subprocess.run`."""
         params = subprocess.list2cmdline(list(map(str, args)))
         command = f"{self._command} {params}".strip()
```

Comment:
One alternative not implemented in this PR is to pass `command` as a list of arguments to `subprocess.run`...
In that case Python should know how to handle whitespaces in the path...

I wonder how backward compatible that would be, and if it would break something in the main repository or in the extensions...

Reply:
I remember that I wrote `shell.py` back then only because I wanted to avoid dependencies. Is there maybe a package that does all this way easier?

Reply:
I think the popular choice in python for running commands/subprocess is [`invoke`](https://github.com/pyinvoke/invoke)... The author of Nox also [recommended it to me some time ago](https://github.com/theacodes/nox/issues/333)
But I have the impression `invoke`'s use case is much more complex...